tutorials/developer-tutorials/createNewIdentity.md

```diff
@@ -0,0 +1,23 @@
+## **Create New Identity Using Local Fabric CA**
+`2-3 mins`
+
+> **Important:** You should have a **smart contract** instantiated on your Local Fabric to follow this tutorial. Follow part 1 of this series first for instructions.
+
+
+## Learning Objectives
+
+* Create a new identity (no attributes) using Local Fabric CA
+
+---
+<details>
+<summary><b>1. Create a new identity</b></summary>
+
+In the very first tutorial (Tutorial 1: Local smart contract development), we used a pre-set identity (`admin`) to connect to the Local Fabric gateway and submit/evaluate transactions. In this tutorial, we will learn how to create _our own_ identity.
+
+Having multiple identities enrolled with the Local Fabric CA (Certificate Authority) allows you to represent different participants in a blockchain network. For example two identities `Jack` (the customer) and `Tesla` (the manufacturer) may be permitted to submit different transactions in a smart contract.
+
+1. In the `FABRIC ENVIRONMENTS` panel, look for `ca.org1.example.com` (it's under Nodes), right click it and choose `Create Identity (register and enroll)`. 
+2. You will be asked to provide a name for your identity. For the purpose of this tutorial, we will call our identity `Jack` and select `No` when asked to add attributes (These will be covered in a later tutorial).
+3. Upon submitting your request, you should see a dialog box at the bottom right of the screen confirming that your identity `Jack` has been created. The newly created identity should also appear in the `Fabric Wallets` panel under `Local Fabric Wallet`. 
+4. To connect to the Local Fabric gateway using this identity, you simply click `Local Fabric` under the Fabric Gateways panel and select the identity you wish to connect with (Jack in our case).
+> Command Palette alternative: `Create Identity (register and enroll)`
```

Comment:
You need a summary at the bottom and also a link to the next tutorial

Reply:
Done.